lambda/script.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+import boto3  # type: ignore
+import googleapiclient.discovery  # type: ignore
+from google.oauth2 import service_account  # type: ignore
```

Comment:
boto3 is a nightmare to mock, but do try to write simple interfaces for other plugins - look at the cyber-security-api for examples of lots of little interfaces :muscle:

Reply:
I think we might have to file googleapi under the same category as boto3 - we've had a look around for pre-typed stuff and had a look into the googleapi docs and it seems like a big and difficult task - what do you think?